demos/browser/server.js

```diff
@@ -19,25 +19,39 @@ console.info('Using index path', indexPagePath);
 
 const indexPage = fs.readFileSync(indexPagePath);
 
-// Create ans AWS SDK Chime object. Region 'us-east-1' is currently required.
+// Create ans AWS SDK Chime object. Region 'us-east-1' is globally available..
 // Use the MediaRegion property below in CreateMeeting to select the region
 // the meeting is hosted in.
 const chime = new AWS.Chime({ region: 'us-east-1' });
+const chimeRegional = new AWS.ChimeSDKMeetings({ region: 'us-east-1' });
 const sts = new AWS.STS({ region: 'us-east-1' })
 
 // Set the AWS SDK Chime endpoint. The global endpoint is https://service.chime.aws.amazon.com.
 const endpoint = process.env.ENDPOINT || 'https://service.chime.aws.amazon.com';
-console.info('Using endpoint', endpoint);
-
+console.info('Using global endpoint', endpoint);
 chime.endpoint = new AWS.Endpoint(endpoint);
 
+if (process.env.ENDPOINT) {
```

Comment:
I don't think we need this for now right? This is mainly to test with gamma endpoint.

Reply:
It's pre-existing functionality for the global endpoint, I just added for the regional one as well. See line #30.

Reply:
It was functionality that was there before (i.e. allowing setting of the ENDPOINT in the demo app).